gravity.sh

```diff
@@ -390,10 +445,53 @@ gravity_DownloadBlocklists() {
 
   if [[ "${status}" -ne 0 ]]; then
     echo -e "\\n  ${CROSS} Unable to copy data from ${gravityDBfile} to ${gravityTEMPfile}\\n  ${output}"
-    return 1
+
+    # Try to attempt a backup restore
+    if [[ -d "${gravityBCKdir}" ]]; then
+      for i in {1..10}; do
+        if try_restore_backup "${i}"; then
+          break
+        fi
+      done
+    fi
+
+    # If none of the attempts worked, return 1
+    if [[ "${i}" -eq 10 ]]; then
```

Comment:
Doesn't this falsely signal backup failed it restoration worked on the 10th backup?

Reply:
https://github.com/pi-hole/pi-hole/pull/5868